extensions/script/ExecuteScript.cpp

```diff
@@ -116,6 +121,9 @@ void ExecuteScript::onTrigger(const std::shared_ptr<core::ProcessContext> &conte
       throw std::runtime_error("No script engine available");
     }
 
+    if (module_directory_.size()) {
+      engine->setModuleDirectories(utils::StringUtils::splitAndTrimRemovingEmpty(module_directory_, ","));
```

Comment:
Not sure if you want to support weird filenames, just want to point out that filenames can start/end with spaces or contain commas.

Reply:
I think regarding spaces, it would be a more common mistake to leave a space at the beginning, the end or between comma-separated entries than using a file path with actual space in it. I'm not sure if it's worth the effort supporting commas in file paths here, if it would ever be a use case maybe it can be requested.